### PR TITLE
[RFR] Add current date to new commits

### PR DESCRIPTION
**Pull request by Kmaschta**

### CODE REVIEW

File: src/git/objects/commits.spec.js [hunk 1]
```diff
@@ -73,4 +74,57 @@ describe('Git Commits', () => {
...
+
+        it('should store the new commit in the store', function* () {
+            const commits = factory(client, repo, store);
+            const commit = yield commits.create({ sha: 'tree sha' }, 'message', { author: 'name' }, 'parents');
+            const storedCommit = store.get(commit.sha);
+
+            assert.deepEqual(commit, storedCommit);
+        });
+
+        it('should standardize thew new commit', function* () {
```
Comment (Kmaschta) [inline src/git/objects/commits.spec.js]: s/thew/the/

File: src/git/objects/commits.js [hunk 2]
```diff
@@ -47,8 +48,7 @@ export default (client, repo, store) => {
             commitMessage: message,
             commitAuthor: {
                 ...author,
-                // @TODO: Set new Date() with the good format
-                date: '2016-07-24T00:49:30+02:00',
+                date: moment().format(),
```
Comment (fzaninotto) [inline src/git/objects/commits.js]: You don't need `moment` for that. `(new Date()).toISOString()` does the same thing.

Comment (Kmaschta) [inline src/git/objects/commits.js]: There are barely different:
```js
(new Date()).toISOString() // 2016-08-25T15:46:19.126Z
moment().format()          // 2016-08-25T17:46:19+02:00
```

According to the [documentation](https://developer.github.com/v3/git/commits/#optional-parameters), Github API needs the second format.

File: src/git/objects/commits.spec.js (hunk 1 above)
Comment (Kmaschta) [inline src/git/objects/commits.spec.js]: s/thew/the/

Comment (Kmaschta) [inline src/git/objects/commits.spec.js]: Sedyception

**Comment by djhi** [comment]
:+1:

**Pull request merged**